android/src/main/java/ly/count/dart/countly_flutter/CountlyFlutterPlugin.java

```diff
@@ -1004,5 +992,21 @@ static void log(String message, Throwable tr, LogLevel logLevel) {
         return list;
     }
 
+    private void populateConfig(JSONObject configJson) throws JSONException {
+        String deviceID = configJson.getString("deviceID");
```

Comment:
this will produce and exception in case of no value, that check should be improved